sdk/node/test/accessTokens.js

```diff
@@ -12,141 +12,141 @@ const client = new chain.Client()
 
 let tokenId, clientTokenId, networkTokenId
 
-describe('Access tokens test', () => {
+function createToken() {
+  tokenId = `token-${uuid.v4()}`
+  return client.accessTokens.create({
+    type: 'client',
```

Comment:
Setting the token type is deprecated, and shouldn't be used except in the deprecated behavior tests.